sound/soc/soc-topology.c

```diff
@@ -1477,8 +1477,6 @@ static int soc_tplg_dapm_widget_create(struct soc_tplg *tplg,
 		goto widget;
 	}
 
-	control_hdr = (struct snd_soc_tplg_ctl_hdr *)tplg->pos;
```

Comment:
@plbossart is the commit in the commit message correct?  I think its Peter's patch that would make this assignment unncessary "ASoC: soc-topology: Move template info print soc_tplg_dapm_widget_create()"

Reply:
it seems correct @ranj063 

Peter's patch does this
````
control_hdr = (struct snd_soc_tplg_ctl_hdr *)tplg->pos;
-	dev_dbg(tplg->dev, "ASoC: template %s has %d controls of type %x\n",
-		w->name, w->num_kcontrols, control_hdr->type);
````

so it leaves this assignment unmodified.

Reply:
right so its Peter's patch that makes this assignment unnecessary no? Before Peter's patch, we still need this assignment fot the dev_dbg() statement to work.

Reply:
Ah yes you're right @ranj063, I'll just add those two commits in the commit message.

Reply:
actually no, it'll become a fixup since Peter's patch is not upstream just yet.